docs/sources/calendar-schedules/ical-schedules/index.md

```diff
@@ -14,52 +14,63 @@ weight: 300
 
 # Import on-call schedules
 
-Use your existing calendar app with iCal format to manage and customize on-call schedules — import rotations and shifts from your calendar app to Grafana OnCall for widely accessible scheduling. iCal imported schedules appear in Grafana OnCall as read-only schedules but can be leveraged similarly to a web-based schedule.
+Use your existing calendar app with iCal format to manage and customize on-call schedules — import rotations and shifts
+from your calendar app to Grafana OnCall for widely accessible scheduling. iCal imported schedules appear in Grafana
+OnCall as read-only schedules but can be leveraged similarly to a web-based schedule.
 
 ## Before you begin
 
-- Verify that your calendar app supports iCal format 
+- Verify that your calendar app supports iCal format
 - Ensure you have the proper permissions in Grafana OnCall
 
 ## Configure an on-call schedule from iCal import
 
 There are three key parts to configuring on-call schedules using iCal import:
+
 1. Create a primary on-call calendar and an optional override calendar in your calendar app.
 1. Import the calendars into Grafana OnCall and configure additional schedule settings.
 1. Link your schedule to corresponding escalation chains for alert notifications to be sent to the proper on-call user.
 
 ### Create your on-call schedule calendar
 
-Create a dedicated calendar to map out your on-call coverage using calendar events. Be sure to take advantage of the features of your calendar app to configure event recurrence, duplicate events, etc. 
+Create a dedicated calendar to map out your on-call coverage using calendar events. Be sure to take advantage of the
+features of your calendar app to configure event recurrence, duplicate events, etc.
 
 >**Note:** The exact steps in this section will vary based on your calendar.
 
 To create an on-call schedule calendar:
 
-1. Create a new calendar in your calendar app, then review and adjust default settings as needed. 
-1. In your new calendar, create events that represent on-call shifts. You must use Grafana usernames as the event title to associate users with each shift.
-1. Once your on-call calendar is complete, go to your calendar settings to locate the secret iCal URL. For example, in a Google calendar, this URL can be found in **Settings** > **Settings for my calendars** > **Integrate calendar** > **Secret address in iCal format**.
+1. Create a new calendar in your calendar app, then review and adjust default settings as needed.
+2. In your new calendar, create events that represent on-call shifts. You must use Grafana usernames as the event title
+3. to associate users with each shift.
+4. Once your on-call calendar is complete, go to your calendar settings to locate the secret iCal URL. For example, in
+5. a Google calendar, this URL can be found in **Settings** > **Settings for my calendars** > **Integrate calendar** >
+6. **Secret address in iCal format**.
```

Comment:
@iskhakov @joeyorlando it looks like the markdown formatting messed up ordered lists starting new lines as another numbered step. Also we use repetitive list numbering so these should remain as:
1.
1.
1.